Sources/MapboxNavigation/Feedback/FeedbackItem.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import MapboxCoreNavigation
+
+/**
+ A single feedback item displayed on an instance of `FeedbackViewController`.
+ */
+public struct FeedbackItem {
+    /**
+     The title of feedback item. This will be rendered directly below the image.
+     */
+    public var title: String
+    
+    /**
+     An image representation of the feedback.
+     */
+    public var image: UIImage
+    
+    /**
+     The type of feedback that best describes the event.
+     */
+    public var type: FeedbackItemType
+    
+    /**
+     Creates a new `FeedbackItem` from active navigation feedback.
+     */
+    public init(title: String, image: UIImage, activeNavigationFeedbackType: FeedbackType) {
+        self.title = title
+        self.image = image
+        self.type = .activeNavigation(activeNavigationFeedbackType)
+    }
+    
+    /**
+     Creates a new `FeedbackItem` from passive navigation feedback.
+     */
+    public init(title: String, image: UIImage, passiveNavigationFeedbackType: PassiveNavigationFeedbackType) {
```

Comment:
Do we really need two separate constructors?

Reply:
Agree, I've united them into one.